code/modules/clothing/gloves/special.dm

```diff
@@ -122,6 +122,7 @@
 	icon_state = "latex_black"
 	inhand_icon_state = "greyscale_gloves"
 	greyscale_colors = "#15191a"
+	clothing_traits = list(TRAIT_QUICK_CARRY, TRAIT_FASTMED)
```

Comment:
I feel like at this point you'd be best served making this a nitrile subtype.

Reply:
Nitrile gloves have quicker_carry, which isn't a very coronerlike aestheticore